subsys/usb/device/class/audio/Kconfig

```diff
@@ -19,4 +19,25 @@ module = USB_AUDIO
 module-str = USB Audio
 source "subsys/logging/Kconfig.template.log_config"
 
+config USB_DEVICE_AUDIO_MIN_VOLUME
+	hex "USB Audio Device Min Volume"
+	default 0
+	range 0 0x7FFF
+	help
+	  attributes can range from 0x7FFF down to 0
```

Comment:
The real purpose behind help text is to allow user to get some idea what the value is all about. Talking about some numbers in context of volume is not giving user real idea of what the numbers signify.

USB Device Class Definition for Audio Devices Release 1.0:
> The settings for the CUR, MIN, and MAX attributes can range from +127.9961 dB (0x7FFF) down to -127.9961 dB (0x8001) in steps of 1/256 dB or 0.00390625 dB (0x0001). The range for the CUR attribute is extended by code 0x8000, representing silence, i.e., -∞ dB. The settings for the RES attribute can only take positive values and range from 1/256 dB (0x0001) to +127.9961 dB (0x7FFF). 

I would expect the help text to at least mention that 0x0100 equals to 1 dB.

Reply:
Thanks very much. 
I Updated this file.